docs/engineering/android-app/configuring/config-types/application.mdx

```diff
@@ -23,7 +23,7 @@ There can only be one instance of application configuration for the entire appli
     ],
     "useDarkTheme": false,
     "syncInterval": 15,
-    "syncStrategy": [
+    "syncStrategies": [
```

Comment:
```suggestion
    "syncStrategy": [
```

Reply:
@dubdabasoduba `ApplicationConfiguration` on main still uses `syncStrategies`. We can confirm what was agreed on whether we can have multiple sync strategies.

Reply:
@qiarie I am going to mark this as resolved since we already have a discussion on going